index.html

```diff
@@ -60,12 +60,7 @@ <h1>CODEVID-19</h1>
             href="https://forms.gle/yHayrwJ2yhGjjTZD6"
             target="_blank"
             class="button primary icon solid fa-external-link-alt"
-            >Vote For a Finalist</a
-          >
-        </li>
-        <li>
-          <a href="{{ '/join-the-fight' | relative_url }}" class="button"
-            >Join the Fight</a
+            >Watch the Final Award Presentation</a
```

Comment:
Maybe "Watch the Final Awards" is enough

Reply:
I can't add a comment on L57 for some reason, but I think I prefer how it looks when `fit` is removed as the class.  I only used `fit` when we had 3-4 buttons.

Reply:
Ok. Why don't I merge and you can tweak the classes as I'm not familiar with what `fit` does.